lib/grammers-session/src/message_box/mod.rs

```diff
@@ -534,30 +561,36 @@ impl MessageBox {
         &mut self,
         difference: tl::enums::updates::Difference,
         chat_hashes: &mut ChatHashCache,
-    ) -> (
-        Vec<tl::enums::Update>,
-        Vec<tl::enums::User>,
-        Vec<tl::enums::Chat>,
-    ) {
+    ) -> Result<
```

Comment:
It seems we always return `Ok`, so this `Result<>`  is redundant (unless I missed a `?`).